cob_obstacle_distance/CMakeLists.txt

```diff
@@ -33,22 +33,25 @@ generate_messages(
 # Declare catkin package
 catkin_package(
    INCLUDE_DIRS include
-   LIBRARIES
+   LIBRARIES parsers
    CATKIN_DEPENDS roscpp sensor_msgs geometry_msgs visualization_msgs message_runtime
-   DEPENDS eigen
+   DEPENDS assimp
 )
 
-include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${FCL_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
+include_directories(include ${catkin_INCLUDE_DIRS} ${EIGEN_INCLUDE_DIRS} ${FCL_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS} ${ASSIMP_INCLUDE_DIRS})
 
 ## Build
+add_library(parsers src/parsers/stl_parser.cpp src/parsers/mesh_parser.cpp)
+target_link_libraries(parsers ${fcl_LIBRARIES} ${catkin_LIBRARIES} ${Assimp_LIBRARIES})
```

Comment:
should this be `ASSIMP_LIBRARIES'?
same for fcl...are those cmake variables actually this inconsitent?